bindings/rust/certs/generate.sh

```diff
@@ -12,3 +12,16 @@ openssl req -x509 -newkey rsa:4096 -keyout key_rsa.pem -out cert_rsa.pem -sha256
 # used for TLS 1.3 connections
 echo "generating ec key and cert"
 openssl req -x509 -newkey ec -pkeyopt ec_paramgen_curve:P-256 -keyout key.pem -out cert.pem -sha256 -days 36500 -nodes -subj "/C=US/ST=AZ/L=Tempe/O=Amazon/OU=AmazonWebServices/CN=localhost"
+
+
+# used for testing IPv6. includes the localhost IPv6 address as a SAN.
```

Comment:
Super nit: see this is the problem with not capitalizing comments, now you've got a missing capital after a period. But if you capitalize that lowercase 'i' your first lowercase 'u' looks silly, but if you capitalize your first lowercase 'u' you don't match the other comments :P

Reply:
True. I updated the other comments in this file so I can capitalize this comment.